src/Storage/RedisStorageAdapter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Opensoft\Rollout\Storage;
+
+/**
+ * Storage adapter using Redis
+ *
+ * @author Woody Gilk <@shadowhand>
+ */
+class RedisStorageAdapter implements StorageInterface
+{
+    /**
+     * @var string
+     */
+    const DEFAULT_GROUP = 'rollout_feature';
+
+    /**
+     * @var object
+     */
+    private $redis;
+
+    /**
+     * @var string
+     */
+    private $group = self::DEFAULT_GROUP;
+
+    public function __construct($redis, $group = null)
```

Comment:
Because predis and native redis use different classes, I cannot type hint this. Duck typing will have to do.